accepted/2020/better-obsoletion/obsoletions-in-net5.md

```diff
@@ -0,0 +1,141 @@
+# Obsoletions in .NET 5
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+For .NET 5, [we're making](better-obsoletion.md) it more viable to obsolete APIs.
+In this document, we're describing which APIs we intend to obsolete in .NET 5 and
+which diagnostic IDs we will use for those.
+
+With `dotnet/runtime` PR [#33248](https://github.com/dotnet/runtime/pull/33248),
+the `ObsoleteAttribute` gained new `DiagnosticId` and `UrlFormat` properties. With
+these properties, obsoletions can be grouped together when multiple APIs are
+marked as obsolete for the same reason. This capability is reflected in the list
+of obsoletions below. All of these obsoletions will use the `UrlFormat` of
+`https://aka.ms/dotnet-warnings/{0}`.
+
+1. The Constrained Execution Region (CER) feature is no longer implemented
+    * DiagnosticId: `MSLIB0004`
+    * APIs:
+        * `System.Runtime.ConstrainedExecution.PrePrepareMethodAttribute`
+        * `System.Runtime.ConstrainedExecution.ReliabilityContractAttribute`
+        * `System.Runtime.ConstrainedExecution.Cer`
+        * `System.Runtime.ConstrainedExecution.Consistency`
+        * `System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup`
+        * `System.Runtime.CompilerServices.RuntimeHelpers.PrepareConstrainedRegions`
+        * `System.Runtime.CompilerServices.RuntimeHelpers.PrepareConstrainedRegionsNoOP`
+        * `System.Runtime.CompilerServices.RuntimeHelpers.ProbeForSufficientStack`
+2. This App Domain method is no longer implemented
+    * DiagnosticId: `MSLIB0005`
+    * APIs:
+        * `PrepareContractedDelegate`
+2. The Global Assembly Cache is no longer implemented
+    * DiagnosticId: `MSLIB0006`
+    * APIs:
+        * `System.Reflection.Assembly.GlobalAssemblyCache`
+3. `System.Net.WebSockets.WebSocket.IsApplicationTargeting45` is no longer applicable
```

Comment:
This is already marked Obsolete.  Are we saying we're just going to give it a diagnostic id / message?

Reply:
Ah, I missed that. It's probably not worth it to add the id/url for this one (as we'd then need to have a page for the URL too).  I'll remove this one.